src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/FolderNameEditor.FolderBrowserStyles.cs

```diff
@@ -2,22 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using static Interop;
-
 namespace System.Windows.Forms.Design
 {
     public partial class FolderNameEditor
     {
         [Flags]
         protected enum FolderBrowserStyles
         {
-            BrowseForComputer = unchecked((int)Shell32.BrowseInfoFlags.BIF_BROWSEFORCOMPUTER),
-            BrowseForEverything = unchecked((int)Shell32.BrowseInfoFlags.BIF_BROWSEFOREVERYTHING),
-            BrowseForPrinter = unchecked((int)Shell32.BrowseInfoFlags.BIF_BROWSEFORPRINTER),
-            RestrictToDomain = unchecked((int)Shell32.BrowseInfoFlags.BIF_DONTGOBELOWDOMAIN),
-            RestrictToFilesystem = unchecked((int)Shell32.BrowseInfoFlags.BIF_RETURNONLYFSDIRS),
-            RestrictToSubfolders = unchecked((int)Shell32.BrowseInfoFlags.BIF_RETURNFSANCESTORS),
-            ShowTextBox = unchecked((int)Shell32.BrowseInfoFlags.BIF_EDITBOX)
+            BrowseForComputer = unchecked((int)PInvoke.BIF_BROWSEFORCOMPUTER),
+            BrowseForEverything = unchecked((int)0x00004000),
+            BrowseForPrinter = unchecked((int)PInvoke.BIF_BROWSEFORPRINTER),
+            RestrictToDomain = unchecked((int)PInvoke.BIF_DONTGOBELOWDOMAIN),
+            RestrictToFilesystem = unchecked((int)PInvoke.BIF_RETURNONLYFSDIRS),
+            RestrictToSubfolders = unchecked((int)PInvoke.BIF_RETURNFSANCESTORS),
+            ShowTextBox = unchecked((int)PInvoke.BIF_EDITBOX)
```

Comment:
To improve readability could you please align all the value? We're generally striving to do this, but in some places we failed to do so.
```suggestion
            BrowseForComputer        = unchecked((int)PInvoke.BIF_BROWSEFORCOMPUTER),
            BrowseForEverything      = unchecked((int)0x00004000),
            BrowseForPrinter         = unchecked((int)PInvoke.BIF_BROWSEFORPRINTER),
            RestrictToDomain = unchecked((int)PInvoke.BIF_DONTGOBELOWDOMAIN),
            RestrictToFilesystem = unchecked((int)PInvoke.BIF_RETURNONLYFSDIRS),
            RestrictToSubfolders = unchecked((int)PInvoke.BIF_RETURNFSANCESTORS),
            ShowTextBox = unchecked((int)PInvoke.BIF_EDITBOX)
```